### PR TITLE
Bump Spring52 and Spring53 to mitigate CVE-2022-22965 

### PR DESCRIPTION
**Pull request by Chromico**
Updating Spring53 and Spring52 due to CVE-2022-22965 

See details: https://tanzu.vmware.com/security/cve-2022-22965

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -329,8 +329,8 @@
         <spring.osgi.version>1.2.1</spring.osgi.version>
         <spring31.version>3.1.4.RELEASE</spring31.version>
         <spring43.version>4.3.30.RELEASE_1</spring43.version>
-        <spring52.version>5.2.13.RELEASE_1</spring52.version>
-        <spring53.version>5.3.14_1</spring53.version>
+        <spring52.version>5.2.20.RELEASE_1</spring52.version>
+        <spring53.version>5.3.18_1</spring53.version>
```
Comment (oscerd) [inline pom.xml:333]: 5.3.18_1 bundle is not available yet from Servicemix.

**Review by oscerd — CHANGES_REQUESTED** [review]
Just one thing.

**Comment by jbonofre** [comment]
I will do the right PR when the bundles will be available.

File: pom.xml (hunk 1 above)
Comment (Chromico) [reply pom.xml:333]: Hopefully, they'll release it soon

**Pull request closed without merging**